include/RouteMap.h

```diff
@@ -373,7 +381,12 @@ struct RouteMapConfiguration {
     // The penalty time to tack the boat, in seconds.
     // The penalty time is added to the route calculation for each tack.
     double TackingTime;
-    // Balance the influence of the wind and the ocean current on the route calculation.
+    // When wind opposes current rough seas can be produced.
+    // This constraint takes the dot product of the current and wind vectors, and if the result exceeds this value,
+    // navigation in this area is avoided.
+    // For example, a value of 60 would avoid 30 knots of wind opposing a 2 knot current as well as
+    // 20 knots of wind opposing a 3 knot current.
+    // Higher values allow for rougher conditions. The special value 0 (default) allows any conditions.
     double WindVSCurrent;
     // The minimum safety distance to land, in nautical miles.
     // The calculated route will avoid land within this distance.
```

Comment:
I'm wondering if the Safety margin is calculated based on the crude GSHHG polygon resolution. I have crude, low, intermediate, high and full resolution installed, and the route calculation does not seem to take into consideration the detailed polygons.

Reply:
It should be using the highest detail...

It would sure be nice if it got depth from the chart.

Reply:
> It would sure be nice if it got depth from the chart.

I'm starting with trivial contributions to understand the code logic, hopefully at some point I'll be able to make more meaningful contributions like this one.

Reply:
I see the following statement should be logged. I will check.
```
  wxLogMessage("GSHHG: Loaded quality %d for land crossing detection.",
               bestQuality);
```

Reply:
After more troubleshooting, I found out the cause is https://github.com/OpenCPN/OpenCPN/pull/4041